price_feed_poller.py

```diff
@@ -26,11 +26,11 @@ def handle_requestUpdate(
 
       if balance == 0:
           raise Exception("Account does not have any funds")
+      print(f"Balance: {balance} wei")
 
       if gas_price is None:
-          gas_price = w3.eth.generateGasPrice()
-
-      print(f"Got {balance} wei")
+            print(f"Estimating gas price from last blocks...")
+        gas_price = w3.eth.generateGasPrice()
```

Comment:
Wtf

Reply:
I mean, doesn't python error on wrong indentation?

Reply:
seems not! :) should be fixed now